src/components/button/Button.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Button from './Button';
+
+describe('<Button />', () => {
+  const btn = mount(<Button label="tes-button"></Button>).find(Button);
```

Comment:
This is picky but I think typo - "test-button" 😬

Reply:
If this is just a temporary file to be used a placeholder, no worries on fixing this

Reply:
It's temporary, but I will fix this anyway 😛 Thanks!!